modules/express/src/clientRoutes.ts

```diff
@@ -1,16 +1,16 @@
 import bodyParser = require('body-parser');
 
-const BitGoJS = require('./index');
-const TransactionBuilder = require('./transactionBuilder');
-const common = require('./common');
+const BitGoJS = require('../../core/src');
+const TransactionBuilder = require('../../core/src/transactionBuilder');
+const common = require('../../core/src/common');
```

Comment:
Gross and icky, but fixing may require refactoring these parts out into a common module. I'm open to suggestions here.